### ISSUE TITLE
Add column "series" to "my videos"

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#530]
Makes sense as it's a very important data point about an event. 

Blocked by #527

**Comment by oas777** [comment]
Is this adding a "column" to 
![Image](https://user-images.githubusercontent.com/1752342/187908525-50ee511e-0e8f-4fae-9fa7-de743219d6aa.png)
and
![Image](https://user-images.githubusercontent.com/1752342/187908857-54aa5e39-462e-473a-8f43-07490da09d61.png)?

**Comment by oas777** [comment]
If it is, please add the other "columns" ("Video hochladen", "Video aufnehmen") on the left and indent them so the hierarchy under "Übersicht" is visible. And absoulutely harmonize the wording:

![Image](https://user-images.githubusercontent.com/1752342/187909487-ea1aaf0d-3291-4df8-9309-bdc0ee896191.png)

**Comment by LukasKalbertodt** [comment]
No. You are probably talking about "my series"? That's tracked by https://github.com/elan-ev/tobira/issues/355

This issue is about the table on the "my videos" page:

![image](https://user-images.githubusercontent.com/7419664/187909490-f1a4f4a9-1c5e-4b2c-883f-ca64b0b40dbc.png)

There, we want to add a column "series" (to the currently existing "title" and "created" columns), showing the title of the series of that event.

**Comment by oas777** [comment]
Ah. That's good. And the series should also be referenced in the event page (https://tobira.opencast.org/~manage/videos/OS0wVcpHWjA).

**Comment by LukasKalbertodt** [comment]
Implemented in #1374
